base-image/Dockerfile

```diff
@@ -121,15 +122,14 @@ ONBUILD RUN echo "Checking for pip 'requirements.txt'..." \
         ; [ -d binder ] && cd binder \
         ; [ -d .binder ] && cd .binder \
         ; if test -f "requirements.txt" ; then \
-        ${NB_PYTHON_PREFIX}/bin/pip install --no-cache-dir -r requirements.txt \
+        && ${NB_PYTHON_PREFIX}/bin/pip install --no-cache-dir -r requirements.txt \
```

Comment:
remove && coming after `then` (similarly for changes below)

Reply:
🤦